res_partner_operating_unit/models/res_users.py

```diff
@@ -2,27 +2,30 @@
 # Copyright (C) 2019 Open Source Integrators
 # Copyright (C) 2020 Serpent Consulting Services
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
-from odoo import _, api, models
+from odoo import Command, _, api, models
 from odoo.exceptions import UserError
 
 
 class ResUsers(models.Model):
     _inherit = "res.users"
 
-    @api.model
-    def create(self, vals):
-        res = super().create(vals)
-        res.partner_id.operating_unit_ids = [(4, res.default_operating_unit_id.id)]
-        res.check_partner_operating_unit()
-        return res
+    @api.model_create_multi
+    def create(self, vals_list):
+        users = super().create(vals_list)
+        for user in users:
+            user.partner_id.operating_unit_ids = [
+                [Command.set(user.default_operating_unit_id.ids)]
```

Comment:
4 is `Command.link`
6 is `Command.set`

Reply:
My bad we want to link. Thanks @vvrossem